assets/prototype/domain/eventEditor/tickets/ticketPriceCalculator/formDecorators/usePriceAmountDecorator.js

```diff
@@ -0,0 +1,30 @@
+import { amountsEqual, getFromFormData } from './utilities';
+import useTicketPriceCalculator from '../hooks/useTicketPriceCalculator';
+
+const usePriceAmountDecorator = () => {
+	const calculator = useTicketPriceCalculator();
+	return {
+		field: /^prices\[\d+\]\.amount$/,
+		isEqual: amountsEqual,
+		updates: (value, name, formData) => {
+			const reverseCalc = getFromFormData('ticket.reverseCalculate', formData);
```

Comment:
can this `'ticket.reverseCalculate'` somehow be typed?

Reply:
just realized that this question is a bit out of the scope of current PR, but worth asking :)

Reply:
ya I have another issue for converting things to TS ==> #2004 
(although I've started to do that a bit already in my next PR as well)